Gemfile

```diff
@@ -9,3 +9,7 @@ if Dir.exist?(logstash_path) && use_logstash_source
   gem 'logstash-core', :path => "#{logstash_path}/logstash-core"
   gem 'logstash-core-plugin-api', :path => "#{logstash_path}/logstash-core-plugin-api"
 end
+
+if RUBY_VERSION == "1.9.3"
+  gem 'rake', '12.2.1'
+end
```

Comment:
what exactly is the purpose of this?

Reply:
@colinsurprenant This is to avoid the intermittent test failures that this gem (and others) have seen in Travis builds against `5.6`, along the lines of 
```
Fetching rake 12.3.1
Installing rake 12.3.1
Gem::RuntimeRequirementNotMetError: rake requires Ruby version >= 2.0.0. The
current ruby version is 1.9.
An error occurred while installing rake (12.3.1), and Bundler
cannot continue.
Make sure that `gem install rake -v '12.3.1' --source 'https://rubygems.org/'`
succeeds before bundling.
```

See https://travis-ci.org/logstash-plugins/logstash-input-sqs/jobs/428789622 for an example

I can move this to a separate commit/PR if this is an issue.

Reply:
I see. What I don't understand is why this is needed in some instances and not for others? For example, https://travis-ci.org/logstash-plugins/logstash-input-snmp passes on 5.6 but does not  have this blurb?

Reply:
I am sorry - I think we've been through that already, but I don't remember the conclusion :P

Reply:
It seems to be intermittent - some kind of Travis weirdness. I also don't remember the conclusion, but I've been adding this snipped into plugins whenever I encounter a build problem.